contract_manager/src/utils/utils.ts

```diff
@@ -0,0 +1,91 @@
+import Web3 from 'web3';
+import evmChainsData from '../../store/chains/EvmChains.json';
+import evmPriceFeedContractsData from '../../store/contracts/EvmPriceFeedContracts.json';
+import evmWormholeContractsData from '../../store/contracts/EvmWormholeContracts.json';
+import * as chains from "viem/chains";
+
+export const getEvmContractAddress = (chainId: number, contractType: string): `0x${string}` => {
+  const chain = evmChainsData.find((c) => c.networkId === chainId);
+  if (!chain) {
+    throw new Error(`Chain with network ID ${chainId} not found`);
+  }
+  
+  if (contractType === 'priceFeed') {
+    const contract = evmPriceFeedContractsData.find((c) => c.chain === chain.id);
+    if (!contract) {
+      throw new Error(`Price feed contract not found for chain ${chainId}`);
+    }
+    return contract.address as `0x${string}`;
+  }
+  
+  if (contractType === 'wormhole') {
+    const contract = evmWormholeContractsData.find((c) => c.chain === chain.id);
+    if (!contract) {
+      throw new Error(`Wormhole contract not found for chain ${chainId}`);
+    }
+    return contract.address as `0x${string}`;
+  }
+  
+  throw new Error(`Unknown contract type: ${contractType}`);
+}
+
+export const getAllEvmChainsIds = () => evmChainsData.map((c) => c.networkId);
```

Comment:
There's no reason to make this a closure, export this as a const.  This takes no arguments and is constant, the closure just means extra cycles every time you use it.